test/nbrowser/ReferenceColumns.ts

```diff
@@ -147,6 +147,23 @@ describe('ReferenceColumns', function() {
         ]
       );
     });
+
+    it('should have linked card for friends', async () => {
```

Comment:
Can you use another document for this test, or create a new one? It is havely used in tests that are not yet part of the grist-core and this additional row breaks them.